blueflood-elasticsearch/src/main/resources/init-es.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#WARNING this will destroy existing ES data and reset it to the proper init state
+function checkFile
```

Comment:
maybe put a little documentation about the script?  or is this in another location?